tests/basic.test.js

```diff
@@ -1,4 +1,4 @@
-const config = require('../')
+const config = require('../packages/eslint-config')
```

Comment:
Can we move tests to the package itself?

Reply:
Yeah surely